proposals/z-index.md

```diff
@@ -0,0 +1,58 @@
+# z-index Convention
+
+## Problem
+
+We have no convention for choosing z-index. This means that developers can set an arbitrary z-index on a component that may have the side effect that it displays above or below another component of which it has no knowledge.
+
+## Solution
+
+### Base layers
+
+First we define base layers in an object.
```

Comment:
Can you enforce here a minimum/standard distance between layer? So this makes easier to add remove value form a layer, for instance we know that we can add until 10 because then we reach the next layer.

Reply:
Can you explain / give an example? The standard distance is 10 in the base layers, not sure what else to put.